pages/profile/myRecipes.js

```diff
@@ -18,7 +21,9 @@ export default function MyRecipes({
   isLoading,
   getRecipeProperty,
   toggleIsFavorite,
+  mutateUser,
 }) {
+  const [addCollection, setAddCollection] = useState(false);
```

Comment:
When working with a boolean, I would expect the state to start with "is" (isLoading, isEditMode, etc.).

I would advise renaming this state to something starting with "is" to make it clear which data type is stored in it